README.md

```diff
@@ -1,5 +1,7 @@
 # npm-login
 
+**Please note: This action is deprecated and should no longer be used. Please, migrate your workflows to the [actions/setup-node](https://github.com/actions/setup-node) action.**
+
```

Comment:
can we show the example you had in the other ticket? I would like people to just have to copy/paste to migrate.